packages/protocol/package.json

```diff
@@ -37,6 +37,7 @@
   "scripts": {
     "build": "tsc -b",
     "clean": "rimraf lib *.tsbuildinfo  coverage .nyc_output ",
+    "generate:index": "glsp generateIndex src -f",
```

Comment:
I'll just add this comment here but in https://github.com/eclipse-glsp/glsp/issues/1286 it was discussed to remove the export of the inversify utilities (or moving them out) due to the creation of the dependency to `reflect-metadata` that it creates. If that is something that we can simply fix by not re-exporting the utilities, I'd suggest to use the same approach as with the test-utils.